.eslintrc

```diff
@@ -11,6 +11,10 @@ plugins:
   - react
   - mozilla
 
+env:
```

Comment:
Not *super* happy about turning on node & browser env for all files, but I think it's probably okay. 

Could use some refinement maybe to use [file glob overrides in .eslintrc](https://eslint.org/docs/user-guide/configuring#configuration-based-on-glob-patterns), but that seemed not to work in the YAML format when I tried. Could also be I didn't have the syntax right.

Reply:
What about just specifying the ESLint environment in the **gulpfile.babel.js** and **webpack.config.js**, a la `/* eslint-env node, browser, whatever */`.
Or if we want to set the node env for a directory, we could just use a nested .eslintrc file (for tests or gulp tasks or whatnot).